getting_started/build-guide/build-from-scratch.rst

```diff
@@ -84,6 +84,10 @@ the newest ALSA from source code.
    ./gitcompile
    sudo make install
 
+.. note::
+
+   | The alsa-lib install operation will override your distro alsa libraries! 
+   | If you don't want to do that then please check "Build SOF with Docker" chapter.
```

Comment:
Probably best to put the warning before the ```make install``` rule otherwise some folks may just copy and paste blindly as they read down and nuke there alsa install. I would probably word it as follows too.
```
**WARNING** installing alsa-lib system wide may break some audio applications. 
Please only do this if you know what you are doing. It's recommended to install it locally (under $HOME)
or use Docker (see "Build SOF with Docker" chapter.)

Reply:
@lgirdwood commit with warning message v2 added